components/suggestion/channel_mention_provider.jsx

```diff
@@ -20,7 +20,13 @@ class ChannelMentionSuggestion extends Suggestion {
         const item = this.props.item;
 
         const channelName = item.channel.display_name;
-        const channelIcon = (<span className='suggestion-list__icon suggestion-list__icon--large'><i className='icon icon--no-spacing icon-globe'/></span>);
+
+        let channelIcon;
+        if (item.channel.type === Constants.OPEN_CHANNEL) {
+            channelIcon = (<span className='suggestion-list__icon suggestion-list__icon--large'><i className='icon icon--no-spacing icon-globe'/></span>);
+        } else if (item.channel.type === Constants.PRIVATE_CHANNEL) {
+            channelIcon = (<span className='suggestion-list__icon suggestion-list__icon--large'><i className='icon icon--no-spacing icon-lock'/></span>);
+        }
```

Comment:
I am not a fan of repeated code-blocks in if-clauses. I would suggest using `clsx` for this specific use-case, since the only difference is the icon class.

But since we are not using `clsx` in the webapp I would suggest something like this:

```suggestion
        const channelIcon = (
            <span className='suggestion-list__icon suggestion-list__icon--large'>
                <i className=`icon icon--no-spacing icon-${item.channel.type === Constants.OPEN_CHANNEL ? 'globe' : 'lock'}`/>
            </span>
        );
```

Reply:
Hi @michelengelen , I already change it.